src/TestFramework/TestFramework/Attributes/TestMethod/GitHubWorkItemAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestTools.UnitTesting;
+
+/// <summary>
+/// GitHubWorkItem attribute; used to specify a GitHub issue associated with this test.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public sealed class GitHubWorkItemAttribute : WorkItemAttribute
```

Comment:
`WorkItem` is supposed to be consumed by AzDO?

Is it reflected in trx? Or how does AzDO consumes it?

Reply:
AFAIK it only appears in trx